### DISCUSSION TITLE
Failed to start

### DISCUSSION THREAD

**Discussion by ririko5834** [chatwoot/chatwoot#3076] [Self hosted]
Hosting it on railway.app, when I visit host:
![image](https://user-images.githubusercontent.com/64729293/134634461-11421219-7e61-464f-a2da-991c2646bf80.png)

Logs: https://pastebin.com/raw/i8x9TNzu

Please help me

**Comment by vishnu-narayanan** [comment]
@ririko5834 Looks like you did not run the migrations before starting the app.  The`db:chatwoot_prepare` rake task prepares the database and should be run before starting the rails app.

**Reply by ririko5834** [reply]
How to run that?

**Reply by ririko5834** [reply]
I have set env RAILS_ENV to `production bundle exec rake db:migrate`
And now I am getting this errors: https://pastebin.com/raw/VjAUchG5

**Comment by NicolasV7** [comment]
Same issue, how u solve it? How can i run the migration?
